### PR TITLE
Changes related to : Attendance service | Configured Kafka Topic and …

### PR DESCRIPTION
**Pull request by vguptatw**
Request you to please review the changes and merge them.

### CODE REVIEW

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml [hunk 1]
```diff
@@ -63,8 +63,43 @@ env: |
...
-  - name: EGOV_WORKFLOW_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: egov-workflow-v2
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_CREATE_TOPIC
+    value: "save-attendance-log"
+  - name: ATTENDANCE_SERVICE_LOG_KAFKA_UPDATE_TOPIC
+    value: "update-attendance-log"
+  - name: ATTENDANCE_SERVICE_REGISTER_KAFKA_CREATE_TOPIC
```
Comment (subhashini-egov) [inline deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml:70]: Remove the service bit from ATTENDANCE_SERVICE_LOG_KAFKA**. Service is understood. Can be just ATTENDANCE_LOG**, ATTENDANCE_REGISTER**.

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml [hunk 2]
```diff
@@ -63,8 +63,43 @@ env: |
...
+    value: "save-attendance"
+  - name: ATTENDANCE_SERVICE_REGISTER_KAFKA_UPDATE_TOPIC
+    value: "update-attendance"
+  - name: ATTENDANCE_SERVICE_STAFF_KAFKA_CREATE_TOPIC
+    value: "save-staff"
+  - name: ATTENDANCE_SERVICE_STAFF_KAFKA_UPDATE_TOPIC
```
Comment (subhashini-egov) [inline deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml:76]: Same comment as above. Remove "SERVICE".

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml [hunk 3]
```diff
@@ -63,8 +63,43 @@ env: |
...
+  {{- if index .Values "attendance-service-register-default-limit" }}
+  - name: ATTENDANCE_SERVICE_REGISTER_DEFAULT_LIMIT
+    value: {{ index .Values "attendance-service-register-default-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "attendance-service-register-search-max-limit" }}
+  - name: ATTENDANCE_SERVICE_REGISTER_SEARCH_MAX_LIMIT
+    value: {{ index .Values "attendance-service-register-search-max-limit" | quote }} 
+  {{- end }}
+  {{- if index .Values "attendance-service-individual-service-integration-required" }}
+  - name: ATTENDANCE_SERVICE_INDIVIDUAL_SERVICE_INTEGRATION_REQUIRED
```
Comment (subhashini-egov) [inline deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml:95]: What does this do? Just a flag to turn on/off, I assume?

**Review by subhashini-egov — APPROVED** [review]
Reviewed. Need to have a review of all application properties once the module is complete.

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml (hunk 1 above)
Comment (vguptatw) [reply deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml]: Word service has been removed

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml (hunk 2 above)
Comment (vguptatw) [reply deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml]: Word service has been removed

File: deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml (hunk 3 above)
Comment (vguptatw) [reply deploy-as-code/helm/charts/digit-works/backend/attendance-service/values.yaml]: Staff / Individual / Document service integration is configuration driven. Will integrate them once services will be available.

**Pull request merged**